src/components/Carousel/Carousel.js

```diff
@@ -52,15 +53,15 @@ const MyCarousel = () => {
   );
 }
 
-function RenderSlides({posts}) {
-  let slides = [];
-  for (let i = 0; i < posts.length; i++) {
-    slides.push(<Slide key={i} index={i}><SlideLocation info={posts[i]}></SlideLocation></Slide>);
-  }
+function RenderSlides({beaches}) {
   return (
     <Slider className={styles.slider}>
-          {slides}
-        </Slider>
+      {
+        beaches.map((beach, index) => 
```

Comment:
i used map instead of the iterative array just because it's cleaner and a more common style. basically the same though